docs/README.vlan.bridging.topo.md

```diff
@@ -289,3 +289,247 @@ vppctl
+
+To start with we can see in the below FDB table there is no Entry added yet on SONiC Control plane whereas few FIB entries are learnt when pings are run across the hosts 1 to 6. Each switch maintains its FIB which contains corresponding MAC address of Hosts learnt on that switch. Lets consider Switch 1 running on sonic-br1 container to test the feature as below,
+
+```
+fdbshow
+```
+
+```
+No.    Vlan    MacAddress    Port    Type
+-----  ------  ------------  ------  ------
+Total number of entries 0
+```
+
+```
+vppctl show l2fib all
+```
+```
+    Mac-Address     BD-Idx If-Idx BSN-ISN Age(min) static filter bvi         Interface-Name        
+ 16:d8:51:a3:16:16    2      2      0/1      -       -      -     -             host-ac2           
+ 16:92:ef:71:03:e6    2      11     0/1      -       -      -     -           host-trunk.20        
+ 46:ee:fd:8b:c0:1e    3      13     0/1      -       -      -     -           host-trunk.30        
+ 1a:1e:10:97:35:a5    1      1      0/1      -       -      -     -             host-ac1           
+ ea:5f:5a:6e:06:5d    3      3      0/1      -       -      -     -             host-ac3           
+ f6:d2:03:81:f1:75    1      9      0/1      -       -      -     -           host-trunk.10        
+L2FIB total/learned entries: 6/9  Last scan time: 0.0000e0sec  Learn limit: 16777216 
+
+```
+For FDB Entry addition on SONiC via config file, an fdbadd.json file with Sample MAC and VLAN ID are created, where one of the entry looks like below, please note for **ADD : "OP" = "SET"**, for **DEL "OP" = "DEL"**
+
+```
+  {
+      "FDB_TABLE:Vlan10:10-10-10-10-10-10": {
+          "port": "Ethernet0",
+          "type": "static"
+      },
+      "OP": "SET"
+  }
+
+  {
+    "FDB_TABLE:Vlan10:10-10-10-10-10-10": {
+        "port": "Ethernet0",
+        "type": "static"
+    },
+    "OP": "DEL"
+  }
+```
+
+To add the entries we created in fdbadd.json onto SONiC FDB, run the below command. The FDB table on SONiC and VPP L2 FIB table should look similar to the below which contains our newly added entries plus dynamically learnt entries on data plane.
+
+```
+swssconfig ./fdbadd.json 
+```
+
+```
+fdbshow
+```
+
+`FDB_TABLE:`
+```
+  No.    Vlan  MacAddress         Port       Type
+-----  ------  -----------------  ---------  -------
+    1      10  10:10:10:10:10:10  Ethernet0  Static
+    2      10  40:40:40:40:40:40  Ethernet3  Static
+    3      10  70:70:70:70:70:70  Ethernet0  Dynamic
+    4      20  20:20:20:20:20:20  Ethernet1  Static
+    5      30  30:30:30:30:30:30  Ethernet2  Static
+    6      30  60:60:60:60:60:60  Ethernet2  Dynamic
+    7      30  50:50:50:50:50:50  Ethernet2  Dynamic
+Total number of entries 7
+```
+```
+vppctl show l2fib all
+```
+`VPP FIB_TABLE :`
+```
+    Mac-Address     BD-Idx If-Idx BSN-ISN Age(min) static filter bvi         Interface-Name        
```

Comment:
This entry is not there in VPP but present in SONIC FIB. Check and correct it if it's output capturing issue
   1      10  10:10:10:10:10:10  Ethernet0  Static